index.bs

```diff
@@ -66,6 +66,88 @@ The primary goal is to make as much of this process as possible consistent betwe
 
 <h2 id="conventions">Conventions</h2>
 
+<h3 id="well-known-intrinsic-objects">Well-Known Intrinsic Objects</h3>
+
+Well-known intrinsics are built-in objects that are explicitly referenced by the algorithms of this specification and which usually have Realm specific identities. Unless otherwise specified each intrinsic object actually corresponds to a set of similar objects, one per Realm.
+
+Within this specification a reference such as %name% means the intrinsic object, associated with the current Realm, corresponding to the name. Determination of the current Realm and its intrinsics is described in ES2015, 8.3. The well-known intrinsics are listed in Table X.
+
+<table>
+  <thead>
+    <tr>
+      <th>Intrinsic Name</th>
+      <th>Global Name</th>
+      <th>Description (<em>non-normative</em>)</th>
+    </tr>
+  </thead>
+  <tr>
+    <td>%Loader%</td>
+    <td>Reflect.Loader</td>
+    <td>The Reflect.Loader constructor (<a href="#loader-objects">...</a>)</td>
+  </tr>
+  <tr>
+    <td>%LoaderPrototype%</td>
+    <td>Reflect.Loader.prototype</td>
+    <td>The initial value of the *prototype* data property of %Loader%</td>
+  </tr>
+  <tr>
+    <td>%BrowserLoader%</td>
+    <td></td>
+    <td>The browser loader constructor (<a href="#browser-loader">...</a>)</td>
+  </tr>
+  <tr>
+    <td>%BrowserLoaderPrototype%</td>
+    <td></td>
+    <td>The initial value of the *prototype* data property of %BrowserLoader%</td>
+  </tr>
+  <tr>
```

Comment:
Given that site-packages are half-baked right now, and slated for the latest stage of the [roadmap](https://github.com/whatwg/loader/blob/master/roadmap.md), I think we should just eliminate the feature completely from the spec for now.